types/ckeditor__ckeditor5-heading/tsconfig.json

```diff
@@ -1,7 +1,6 @@
 {
     "compilerOptions": {
         "module": "commonjs",
-        "target": "es6",
```

Comment:
why this change?

Reply:
It was suggested to me in another PR since these files are not expected to be compiled.